npm/javy-cli/index.js

```diff
@@ -150,10 +150,11 @@ function getArgs() {
 	return args;
 }
 
-async function buildBinary() {
+async function buildBinary(version) {
 	const repoDir = cacheDir("build", NAME);
 	try {
 		console.log(`Downloading ${NAME}'s source code...`);
+		fs.rmSync(repoDir, { recursive: true });
 		childProcess.execSync(
 			`git clone https://github.com/${REPO} ${repoDir}`
```

Comment:
This ignores the version argument which is probably incorrect. Arguably the right thing to do is add support in the code for running a plain `javy` executable instead of including a version number. I didn't want to spend more time doing that given I'm not sure how frequently this is used in practice, if at all. We can opt to fix if someone complains.